### ISSUE TITLE
QSAR Examples?

### ISSUE THREAD

**Issue by naeemmrz** [kexinhuang12345/DeepPurpose#139]
I might be blind but I couldn't find any QSAR tutorials among the Jupyter Notebooks, am I looking at the wrong place? I've trying to test deep learning models in predicting the potency of drugs i.e., I have a few thousand smiles and their pIC50 and want to build a QSAR model. I tried some RDKit/related descriptor/fingerprints and was wondering if DeepPurpose had similar applications. Thanks in advance :)

**Comment by kexinhuang12345** [comment]
Definitely, check out this one: https://github.com/kexinhuang12345/DeepPurpose/blob/master/Tutorial_2_Drug_Property_Pred_Assay_Data.ipynb

**Comment by naeemmrz** [comment]
@kexinhuang12345 Thank you!! 

I've tried it out and indeed it provided some improvement over classical ML algorithms, despite using the default hyperparameters for the most part. This might be off-topic/not an issue, what advice would you give for hyperparameter tuning for a regression task? the list is quite large, any suggestions on what hyperparameters would mostly like to significantly impact the model's performance? I started playing with the layer size/number but any advice would be welcome :).

Thanks again.

**Comment by kexinhuang12345** [comment]
Great! yeah, i think it highly depends on the molecule model you are looking at. I think in general, play around with layer size, number, learning rate are important. We also provide a tutorial to use hyperparameter tuning: https://github.com/kexinhuang12345/DeepPurpose/blob/master/DEMO/Drug_Property_Pred-Ax-Hyperparam-Tune.ipynb

**Comment by kexinhuang12345** [comment]
closing for now!
